server/tests/util/integrationTestData.ts

```diff
@@ -74,14 +74,16 @@ interface TestSiteData {
   siteId?;
   city?: string;
   siteName?: string;
+  healthAuthority?: string;
   [key: string]: string | number | boolean | undefined;
 }
 
 export const makeTestSite = async (
-  { siteId, city, siteName, ...rest }: TestSiteData = {
+  { siteId, city, siteName, healthAuthority = 'Vancouver Island', ...rest }: TestSiteData = {
```

Comment:
What's happening here?

Reply:
This is just for when the backend tests make test sites. I wanted to set a default for which health authority a site is in rather than defining it in all tests, and then tests on their own can set it themselves if needed (as seen on the test I added)